doc/assets.md

```diff
@@ -10,41 +10,37 @@ For assets, a special **`ezdesign` asset package** is available.
 <img src="{{ asset("images/foo.png", "ezdesign") }}" alt="foo"/>
 ```
 
-Using `ezdesign` package will resolve current design with theme fallback.
+Using the `ezdesign` package will resolve current design with theme fallback.
 
 By convention, an asset theme directory can be located in:
-* `<bundle_directory>/Resources/public/themes/`
-* `web/assets/themes/`
+- `<bundle_directory>/Resources/public/themes/`
+- `web/assets/themes/`
 
 Typical paths can be for example:
-* `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of `foo` theme.
-* `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of `bar` theme.
-* `web/assets/themes/biz/` => Assets will be part of `biz` theme.
+- `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of the `foo` theme.
+- `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of the `bar` theme.
+- `web/assets/themes/biz/` => Assets will be part of the `biz` theme.
 
-It is also possible to use `web/assets` as a global override directory. 
-If called asset is present **directly under this directory**, it will always be considered first.
+It is also possible to use `web/assets` as a global override directory. If called asset is present **directly under this directory**, it will always be considered first.
 
-> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are
-> *installed* into the `web/` directory.
+> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are *installed* into the `web/` directory.
 
 ## Fallback order
-Default fallback order is the following:
-* Application assets directory: `web/assets/`
-* Application theme directory: `web/assets/themes/<theme_name>/`
-* Bundle theme directory: `web/bundles/<bundle_directory>/themes/<theme_name>/`
+
+The default fallback order is:
+- Application assets directory: `web/assets/`
+- Application theme directory: `web/assets/themes/<theme_name>/`
+- Bundle theme directory: `web/bundles/<bundle_directory>/themes/<theme_name>/`
 
 Calling `asset("js/foo.js", "ezdesign")` can for example be resolved to `web/bundles/app/themes/my_theme/js/foo.js`.
 
-## Performance & assets resolution
-When using themes, paths for assets are resolved at runtime. This is due to how Symfony Asset component is integrated
```

Comment:
> sticking to a fixed width is going to cause problems when future changes to doc files inevitably occur.

It's not a "fixed" length, it's a maximum to be respected within reason.
I still do not see why using unique lines makes things easier, and I have given arguments about why.